### DISCUSSION TITLE
Elevate update not available

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#331] [Q&A] [answered]
There is a new update available for the OAM map theme Elevate but in the Cruiser download manager it is not offered.

**Comment by kds112** [comment]
I am using Cruiser 3.0.32

**Comment by devemux86** [comment]
@kds112 thanks for the report.

Indeed, I don't see the latest version of Elevate on the OpenAndroMaps server.

I contacted the OpenAndroMaps team.

**Comment by devemux86** [comment, marked as answer]
The latest Elevate theme is now on OpenAndroMaps server
and should be available in Cruiser download manager too.

**Reply by kds112** [reply]
Thanks a lot!!

**Reply by devemux86** [reply]
I was informed that there is an automatic synchronization every day.
(they had some maintenance work lately)
